test/base-commands/twilio-client-command.test.js

```diff
@@ -32,69 +48,68 @@ describe('base-commands', () => {
-    });
+    setUpTest()
+      .it('should not allow construction of the base class', async ctx => {
+        expect(() => new TwilioClientCommand([], ctx.fakeConfig)).to.throw(TypeError);
+      });
 
-    setUpTest(['-l', 'debug'], () => 0).it('should fail for a non-existent default project', async ctx => {
-      ctx.testCmd.exit = sinon.fake();
-      await ctx.testCmd.run();
-      expect(ctx.stderr).to.contain('No project configured');
-      expect(ctx.stderr).to.contain('To add project, run: twilio projects:add');
-      expect(ctx.stderr).to.contain('TWILIO_ACCOUNT_SID');
-      expect(ctx.testCmd.exit.calledWith(1)).to.be.true;
-    });
+    setUpTest(['-l', 'debug'])
+      .it('should create a client for the active project', async ctx => {
+        expect(ctx.stderr).to.contain('MyFirstProject');
+        expect(ctx.testCmd.twilioClient.accountSid).to.equal(constants.FAKE_ACCOUNT_SID);
+        expect(ctx.testCmd.twilioClient.username).to.equal(constants.FAKE_API_KEY);
+        expect(ctx.testCmd.twilioClient.password).to.equal(constants.FAKE_API_SECRET + 'MyFirstProject');
+        expect(ctx.testCmd.twilioClient.region).to.equal(undefined);
+      });
 
-    setUpTest(['-p', 'alt', '-l', 'debug']).it('should fail for a non-existent project', async ctx => {
-      ctx.testCmd.exit = sinon.fake();
-      await ctx.testCmd.run();
-      expect(ctx.stderr).to.contain('No project configured');
-      expect(ctx.stderr).to.contain('To add project, run: twilio projects:add -p alt');
-      expect(ctx.stderr).to.contain('TWILIO_ACCOUNT_SID');
-      expect(ctx.testCmd.exit.calledWith(1)).to.be.true;
-    });
+    setUpTest(['-l', 'debug'], { setUpUserConfig: () => 0 })
+      .exit(1)
+      .it('should fail for a non-existent active project', async ctx => {
+        expect(ctx.stderr).to.contain('No project configured');
+        expect(ctx.stderr).to.contain('To add project, run: twilio projects:add');
+        expect(ctx.stderr).to.contain('TWILIO_ACCOUNT_SID');
+      });
 
-    setUpTest(['-p', 'twilio-cli-unit-testing'], userConfig => {
-      userConfig.addProject('twilio-cli-unit-testing', constants.FAKE_ACCOUNT_SID, 'stage');
-    }).it('should create a client for a non-default project', async ctx => {
-      await ctx.testCmd.run();
-      expect(ctx.testCmd.twilioClient.accountSid).to.equal(constants.FAKE_ACCOUNT_SID);
-      expect(ctx.testCmd.twilioClient.username).to.equal(constants.FAKE_API_KEY);
-      expect(ctx.testCmd.twilioClient.password).to.equal(constants.FAKE_API_SECRET + 'twilio-cli-unit-testing');
-      expect(ctx.testCmd.twilioClient.region).to.equal('stage');
-    });
+    setUpTest(['-p', 'alt', '-l', 'debug'])
+      .exit(1)
+      .it('should fail for a non-existent project', async ctx => {
+        expect(ctx.stderr).to.contain('No project configured');
+        expect(ctx.stderr).to.contain('To add project, run: twilio projects:add -p alt');
```

Comment:
nit: To add project -> To add a project

Reply:
Depending on the context they're either adding _a_ project or _the_ project. I'll think about it, but outside the scope of this review.